config/migrations/cockroachdb/5_constraints_with_description.up.sql

```diff
@@ -0,0 +1,2 @@
+-- Add description column to constraints
+ALTER TABLE constraints ADD COLUMN description TEXT NOT NULL DEFAULT 'desc';
```

Comment:
we dont want to default `desc` as that would make it non-optional as `desc` would be inserted for every row

we don't have defaults for any of the other optional description fields (ie: https://github.com/flipt-io/flipt/blob/c77b801c33cecaea14576e68dbf1584af060670b/config/migrations/sqlite3/0_initial.up.sql#L13)

Reply:
Talked offline, and we will just make description nullable.